src/FSharpLint.Core/Text.resx

```diff
@@ -348,4 +348,7 @@
   <data name="RulesAvoidSinglePipeOperator" xml:space="preserve">
     <value>Can use normal function call (instead of pipe operator) because there is no invocation chain (single invocation only).</value>
   </data>
-</root>
+  <data name="RulesSuggestUseAutoProperty" xml:space="preserve">
+    <value>Consider giving auto-properties via the "val" keyword</value>
```

Comment:
@webwarrior-ws please be consistent with other messages above, which add a period at the end

Reply:
done